src/lib/y2storage/proposal/devicegraph_generator.rb

```diff
@@ -87,6 +88,17 @@ def devicegraph(volumes, initial_graph, disk_analyzer)
 
     protected
 
+      # Modifies planned volumes adding an encryption password.
+      # Only volumes that can_live_on_logical_volume could have
+      # an encryption password.
+      # @param volumes [PlannedVolumesList]
+      def add_encryption_password(volumes)
+        return volumes unless settings.use_encryption
+        volumes.select(&:can_live_on_logical_volume).map do |volume|
```

Comment:
That would mean, of course, adapting the wording in the corresponding tests and, as a result, adapting https://github.com/yast/yast-storage-ng/blob/master/doc/boot-requirements.md

I would probably go one step further than adapting the wording. Right now we usually have

*  With a partitions-based proposal
*  With a LVM-based proposal

I would add a third branch all over those tests, leaving it like

* With a plain partitions-based proposal
* With a LVM-based proposal
* With an encrypted proposal

I know the expectations for the last two cases will be almost identical everywhere, but I believe it's worth having explicit separate contexts from the good testing point of view.